src/explore-education-statistics-admin/src/pages/release/components/ReleaseStatusForm.tsx

```diff
@@ -99,6 +99,38 @@ const ReleaseStatusForm = ({
     useToggle(false);
   const [showScheduleErrorModal, toggleScheduleErrorModal] = useToggle(false);
 
+  const submitButtonRef = useRef<HTMLButtonElement>(null);
+
+  const closeConfirmScheduleModal = () => {
+    toggleConfirmScheduleModal.off();
+    setTimeout(() => {
+      submitButtonRef.current?.focus();
+    }, 1);
+  };
```

Comment:
Is there a reason we're avoiding the `useEffect` on `showScheduleErrorModal` like we've done elsewhere?
```tsx
useEffect(()=>{
  if(!showScheduleErrorModal) { submitButtonRef.current?.focus(); }
},[showScheduleErrorModal, submitButtonRef]);
```